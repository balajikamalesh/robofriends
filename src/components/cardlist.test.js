import {shallow, render, mount} from 'enzyme';
import React from 'react';
import Cardlist from './cardlist';

it('expect to render card component',() => {
    const mockRobots = [
        {
            id: 1,
            name: 'balaji',
            email: 'balaji@gmail.com'
        },
        {
            id: 2,
            name: 'kamalesh',
            email: 'kamalesh@gmail.com'
        }
    ]
    expect(shallow(<Cardlist robots={mockRobots}/>)).toMatchSnapshot();
})