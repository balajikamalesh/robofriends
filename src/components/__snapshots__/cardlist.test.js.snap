// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card component 1`] = `
<div>
  <Card
    email="balaji"
    id={1}
    key="1"
    name="balaji"
  />
  <Card
    email="kamalesh"
    id={2}
    key="2"
    name="kamalesh"
  />
</div>
`;
